app/mailers/rents_mailer.rb

```diff
@@ -1,15 +1,26 @@
 class RentsMailer < ApplicationMailer
   def send_new_rent_email(rent_id)
-    @rent = Rent.find(rent_id)
-    @user = @rent.user
-    @book = @rent.book
+    get_rental_data(rent_id)
```

Comment:
Avoid using getters and setters in the names :) (https://github.com/rubocop-hq/ruby-style-guide#accessor_mutator_method_names)